starter-core/src/main/java/io/micronaut/starter/feature/build/gradle/templates/buildGradle.rocker.raw

```diff
@@ -527,13 +527,17 @@ java {
 }
 
 @if (features.language().isJava()) {
-tasks.withType(JavaCompile){
+tasks.withType(JavaCompile) {
     options.encoding = "UTF-8"
     options.compilerArgs.addAll([
         '-parameters',
         // enables incremental compilation
         '-Amicronaut.processing.incremental=true',
-        '-Amicronaut.processing.annotations=@project.getPackageName().*'
+        '-Amicronaut.processing.annotations=@project.getPackageName().*',
+      @if (features.contains("graalvm")) {
```

Comment:
I think we can remove these checks for graalvm, this data could be useful for other processors

Reply:
Done